Sources/TestsCommon/APMUtils.swift

```diff
@@ -81,13 +81,11 @@ public class TestCommandMonitor: CommandEventHandler {
 }
 
 public enum EventType: String, Decodable {
-    case commandStartedEvent, commandSucceededEvent, commandFailedEvent,
-         connectionCreatedEvent, connectionReadyEvent, connectionClosedEvent,
-         connectionCheckedInEvent, connectionCheckedOutEvent, connectionCheckOutFailedEvent,
-         poolCreatedEvent, poolReadyEvent, poolClearedEvent, poolClosedEvent,
-         topologyDescriptionChanged, topologyOpening, topologyClosed, serverDescriptionChanged,
-         serverOpening, serverClosed, serverHeartbeatStarted, serverHeartbeatSucceeded,
-         serverHeartbeatFailed
+    case commandStartedEvent, commandSucceededEvent, commandFailedEvent, connectionCreatedEvent, connectionReadyEvent,
+         connectionClosedEvent, connectionCheckedInEvent, connectionCheckOutStartedEvent, connectionCheckedOutEvent,
```

Comment:
only actual change here mod reorganizing lines was adding `connectionCheckOutStartedEvent` which one of the tests I synced uses